src/WebApp/Controllers/RentalCarManagementController.cs

```diff
@@ -88,6 +88,28 @@ public async Task<IActionResult> Register([FromForm] RentalCarManagementNewViewM
             return View("New", inputModel);
         }
     }
+    
+    [HttpPost]
+    public async Task<IActionResult> Delete(string rentalCarId)
+    {
+        return await _resiliencyHelper.ExecuteResilient(async () =>
+        {
+            try
+            {
+                await _rentalCarManagementApi.DeleteReview(rentalCarId);
+                return RedirectToAction("Index");
+            }
+            catch (ApiException ex)
+            {
+                if (ex.StatusCode == HttpStatusCode.NotFound)
+                    return NotFound("Rental car not found.");
+                else if (ex.StatusCode == HttpStatusCode.Conflict)
+                    return Conflict("Unable to delete rental car due to a conflict.");
```

Comment:
Dit is if else stacking, oftwel de "else" is overbodig

Reply:
Gefixt